### PR TITLE
Update prompt.py to clarify how to call commands

### PR DESCRIPTION
**Pull request by richbeales**
### Background

As discussed on the call recently, we tell the LLM to exclusively use the commands in double quotes, but the commands aren't listed in double quotes, and are underscore-separated and not space-separated.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by k-boikov** [comment]
#RichTheLightning
LGTM

**Comment by Boostrix** [comment]
nice, I've seen code reviews here where people pointed out discrepancies in that area.

the command string should probably be internally validated at the command mgr level using a regex to show a warning, especially since commands are likely to be added by external means, like plugins.

File: autogpt/prompts/prompt.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def build_default_prompt_generator() -> PromptGenerator:
     )
     prompt_generator.add_constraint("No user assistance")
     prompt_generator.add_constraint(
-        'Exclusively use the commands listed in double quotes e.g. "command name"'
```
Comment (ntindle) [inline autogpt/prompts/prompt.py:41]: Upload cassettes (or I can)

**Review by ntindle — DISMISSED** [review]

File: autogpt/prompts/prompt.py (hunk 1 above)
Comment (richbeales) [reply autogpt/prompts/prompt.py:41]: I'm happy to  but not sure how?
Can you point me in a direction?

Comment (k-boikov) [reply autogpt/prompts/prompt.py:41]: Done.

**Review by k-boikov — APPROVED** [review]

**Comment by Boostrix** [comment]
FYI: I've seen fewer command hallucinations by adapting the prompt generator to encourage the LLM to respond within the bounds of the list of commands by also adding the length of the list to the prompt itself (aka "pick one between 1-21").

**Pull request merged**